[No-QA] Update the guide to include payment requesting and refund process (#89314)
Expensify/App: contributingGuides/PAYMENT_VIA_EXPENSIFY.md

    - You can do this in **oldDot** by navigating to: Settings > Account > Wallet > Add Personal Bank Account > Connect manually.
  - Skydo is a service that contributors in India have used [Slack Thread](https://expensify.slack.com/archives/C02NK2DQWUX/p1761116747758089).
- View [Enable Global Reimbursement](https://help.expensify.com/articles/expensify-classic/bank-accounts-and-payments/payments/Enable-Global-Reimbursement) to get more details on setting up reimbursements in United States (USD), Canada (CAD), United Kingdom (GBP), European Union (EUR), Australia (AUD) and Singapore (SGD) currencies.
- View [Receive Payments](https://help.expensify.com/articles/expensify-classic/bank-accounts-and-payments/payments/Receive-Payments) for details on receiving payments in the US and internationally, including via Wise (formerly TransferWise), Paypal, Venmo and more.  

After approval it can take between 1 business day and a week, depending on the country and service you use.  

## Requesting Payments

> Request payment, by submitting your reports. in the same month the payment summary is posted. If that is not possible, request it as soon as possible after month-end. This is a hard requirement. If we were to have several contributors submitting high volumes of payments outside of the reporting quarter our financial statements will not be accurate.

Reviewer: ```suggestion
Request payment, by submitting your reports, in the same month the payment summary is posted. If that is not possible, request it as soon as possible after month-end. This is a hard requirement.
```

> Include in the Description field a link to the payment confirmation posted by the BZ team member on the issue for which payment is due. Please do not add any additional details.  

Reviewer: ```suggestion
Include in the Description field a link to the payment confirmation posted by the Expensify employee for which payment is due. Please do not add any additional details.  
```

**Do not ever request payment before an employee has posted a payment summary.** 

**Important!:** If you’re new to NewDot payments, please double check your eligibility date against the date you were assigned to a GH issue.  If you were assigned to a GH issue before your eligibility date, you must be paid via Upwork.

> Once a PR has been deployed to production, those paid via NewDot can request payment, they add ⏸️ emoji in their payment request to denote the payment isn't due now but when the payment summary is posted after the 7-day regression period.

Reviewer: ```suggestion
```

Reviewer: I'm unsure if this might not be best for the folks who approve payments so I'd rather not have it public, for now.
